### ISSUE TITLE
Test Peak create

### ISSUE THREAD

**Issue by HansVraga** [USGS-WiM/STNWeb#225]
Testing (prior to new PostgrSQL tweaks) revealed some strange behavior with creating a peak where it would disappear from the site page.

**Comment by HansVraga** [comment]
Feedback from a user:

I successfully added a peak at one time, but when I got out of this site and then back in the peak was no longer there. I also tried to add a peak from a HWM and was not successful in storing this as a peak.

**Comment by troddick** [comment]
Interesting find - If you create a Peak Summary at a site for an Event (you must have an event chosen to make a peak), and do not choose to use any datafiles or hwms.. just populate the fields and hit create, It will create it. If you leave and come back, it will not be there because...
There is nothing connecting this Peak to this event because they did not choose a data file or hwm. 
See image: If I don't check that hwm, then this Peak will never show up because I didn't link it to the event through the hwm.
![image](https://cloud.githubusercontent.com/assets/1580076/15727039/c37e91d0-2819-11e6-8303-7aba904b1732.png)

Side note: I think if checking that I want to use this HWM as primary (that box with a check mark in it on the right), this should make the hwm checkbox on the left checked -- it doesn't now.

**Comment by troddick** [comment]
As long as I choose a data file and/or hwm to use for interpretation, I can still see the Peak when leaving and coming back. It was created successfully. I went to the map tab, then to the approval tab, then used the Back browser button to get back to the site Dashboard and it's still there. I went to the map, clicked on the site and then the site_no to go to the dashboard and it was still there. I also searched for it and was sent to the dash and it was still there. 
Maybe we should enforce they choose at least 1 hwm or datafile to link this peak to the event.??
